peachjam/templates/peachjam/saved_document_create.html

```diff
@@ -0,0 +1,29 @@
+{% load i18n %}
+<div id="saveDocumentWrapper">
+  <button id="saveDocumentButton"
+          class="btn btn-outline-primary btn-shrink-sm ms-2"
+          type="button"
+          data-bs-toggle="modal"
+          data-bs-target="#saveDocumentModal"
+          aria-expanded="false"
+          hx-post="{% url 'saved_document_create' %}?doc_id={{ form.document.value }}"
+          hx-target="#saveDocumentWrapper">
+    {% trans 'Save document' %}
+  </button>
+  <div hx-swap-oob="true" id="saveDocumentModalDialog" class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <p class="h5 modal-title">{% trans 'Save document' %}</p>
+        <button type="button"
+                class="btn-close"
+                data-bs-dismiss="modal"
+                aria-label="Close"></button>
+      </div>
+      <div class="modal-body">
+        Saving...
```

Comment:
translateable

Reply:
one day I will remember to be translating my strings :sweat_smile:

Reply:
done